Sources/Components/BottomSheet/Transition/BottomSheetGestureController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright © 2018 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+protocol BottomSheetGestureControllerDelegate: class {
+    func gestureDidBegin() -> CGFloat // Expects to get the current position of the bottom sheet
```

Comment:
We have a convention for delegate methods to be in format like 
```swift
func bottomSheetGestureControllerDidBeginGesture(_ controller: BottomSheetGestureController)
```

Reply:
I'm too lazy 😊

Reply:
I'll fix though!